src/zenml/utils/pipeline_docker_image_builder.py

```diff
@@ -638,8 +638,12 @@ def _generate_zenml_pipeline_dockerfile(
         lines.append("RUN chmod -R a+rw .")
 
         if docker_settings.user:
-            lines.append(f"USER {docker_settings.user}")
+            # Switch to root user to change file ownership
+            lines.append("USER root")
```

Comment:
Isn't this redundant since `root` is the default? And even if you were not root at this point, wouldn't `RUN chmod -R a+rw .` already have failed before even reaching this?